Makefile

```diff
@@ -58,6 +58,7 @@ check-venv:
 	fi
 
 install: venv-create
+    . $(VENV_ACTIVATE_FILE); pip install --upgrade pip
```

Comment:
The indentation looks a bit odd. Tabs are needed for `make` targets, and I think this line doesn't have a tab character like the rest. Let's replace it with a tab.

Reply:
Done